icons/map-minus.json

```
@@ -0,0 +1,24 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "colebemis",
+    "karsa-mistmere",
+    "ericfennis",
+    "MarianoFranzese",
+    "jguddas"
+  ],
+  "tags": [
+    "location",
+    "navigation",
+    "travel",
+    "drop",
+    "delete",
+    "remove",
+    "erase"
+  ],
+  "categories": [
+    "location",
+    "navigation",
+    "travel"
+  ]
```

`location` category does not exist. 
```suggestion
  "categories": [
    "navigation",
    "travel"
  ]
```